tools/custom-checks/check-dependency-version-consistency.ts

```diff
@@ -0,0 +1,15 @@
+import { CDVC } from "check-dependency-version-consistency";
+
+export async function checkDependencyVersionConsistency(): Promise<
+  [boolean, string[]]
+> {
+  const errors: string[] = [];
+  const cdvc = new CDVC(process.cwd(), {
+    fix: false,
+  });
+  errors.push(cdvc.toMismatchSummary());
+  return [errors.length === 0, errors];
+}
+
+const dependencyMismatchSummary = await checkDependencyVersionConsistency();
+console.log(dependencyMismatchSummary[1][0]);
```

Comment:
@jagpreetsinghsasan This code will run every time the file gets imported. Please use the pattern established in the other similar scripts instead where we have a main function for logic like this and the main function gets invoked on condition of the script not having been imported by another script but instead having been ran from a shell.

Reply:
@jagpreetsinghsasan Thank you!